packages/fields/README.md

```diff
@@ -56,6 +56,7 @@ Fields share some standard configuration options.
 | `defaultValue` | `Boolean` \| `Function`             | `undefined` | A default value of the field.                                                           |
 | `isUnique`     | `Boolean`                           | `false`     | Whether or not the field should be unique.                                              |
 | `isRequired`   | `Boolean`                           | `false`     | Whether or not the field should be mandatory.                                           |
+| `isOrderable`  | `Boolean`                           | `true`      | Whether or not the field can be ordered in the AdminUI.                                 |
```

Comment:
should this be part of `adminConfig`, I have always been in favor of merging config and adminConfig

Reply:
is the Relationship field orderable by default?

Reply:
Currently all fields are orderable by default

Reply:
Changed this so that default is not orderable, and made this an internal property